### ISSUE TITLE
Calling unknown method: rias\scout\jobs\MakeSearchable::getelement()

### ISSUE THREAD

**Issue by mark-chief** [riasvdv/craft-scout#120]
Hi, 

I have updated to Scout v2.1.2 and I am now getting a load of failed tasks. They all show an error of:

`Calling unknown method: rias\scout\jobs\MakeSearchable::getelement()`

Thanks

**Comment by zizther** [comment]
+1 on this issue.
When saving an entry with Scout 2.1.2 in Craft 3.3.15 it shows this error

```
2019-11-20 16:12:30 [-][1][-][error][craft\queue\QueueLogBehavior::afterError]  [1872] Indexing “Decent Packaging” in “dev_Directory” (attempt: 1) - Error (time: 0.188s): Calling unknown method: rias\scout\jobs\MakeSearchable::getelement()
2019-11-20 16:12:30 [-][1][-][error][yii\base\UnknownMethodException] yii\base\UnknownMethodException: Calling unknown method: rias\scout\jobs\MakeSearchable::getelement() in .../vendor/yiisoft/yii2/base/BaseObject.php:222
```

**Comment by riasvdv** [comment]
Can you post the full stacktrace of the error?

**Comment by mustasj** [comment]
I´m getting the same error

**Comment by riasvdv** [comment]
Fixed in 2.1.3
